pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "torch>=2.2.0",
     "lightning==2.3.0.dev20240328",
     "jsonargparse[signatures]>=4.27.6",
+    "litserve",   # imported by litgpt.deploy
```

Comment:
we might need to make sure we pin the version here, things will move fast on the `litserve` side